smps.proto

```diff
@@ -15,34 +14,45 @@ message PerformanceTestConfig {
             tcp = 1;
             grpc = 2;
```

Comment:
We can toss in udp here. Let’s make a distinction between http and tcp, too.

Reply:
Yes. We do have separate ones for TPC and HTTP right?